### PR TITLE
Change MapLayerImageryProvider.supportsMapFeatureInfo from alpha to public

### PR DESCRIPTION
**Pull request by wilmaier**
This getter is needed in @itwin/map-layers package of viewer-components-react to show the map feature info tool only when a provider is registered to support it.

### CODE REVIEW

File: core/frontend/src/tile/map/MapLayerImageryProvider.ts [hunk 1]
```diff
@@ -67,7 +67,7 @@ export abstract class MapLayerImageryProvider {
   /** @internal */
   public get status() { return this._status; }
 
-  /** @alpha */
+  /** @beta */
   public get supportsMapFeatureInfo() { return false; }
```
Comment (pmconne) [inline core/frontend/src/tile/map/MapLayerImageryProvider.ts:71]:
```suggestion
  public get supportsMapFeatureInfo(): boolean { return false; }
```

File: core/frontend/src/tile/map/MapLayerImageryProvider.ts [hunk 2]
```diff
@@ -67,7 +67,7 @@ export abstract class MapLayerImageryProvider {
   /** @internal */
   public get status() { return this._status; }
 
-  /** @alpha */
+  /** @beta */
```
Comment (pmconne) [inline core/frontend/src/tile/map/MapLayerImageryProvider.ts:70]: Document this. What's it for, why do I need it?

File: core/frontend/src/tile/map/MapLayerImageryProvider.ts (hunk 1 above)
Comment (wilmaier) [reply core/frontend/src/tile/map/MapLayerImageryProvider.ts]: Was asked to make this public instead of beta now

**Review by pmconne — APPROVED** [review]

**Comment by wilmaier** [comment]
@Mergifyio backport release/4.8.x

**Pull request merged**